packages/ckeditor5-engine/src/view/placeholder.ts

```diff
@@ -299,6 +308,19 @@ function getChildPlaceholderHostSubstitute( parent: Element ): Element | null {
 	return null;
 }
 
+/**
+ * Displays a deprecation warning message in the console, but only once per page load.
+ */
+function showDeprecationWarning() {
+	if ( !hasDisplayedWarning ) {
+		console.warn( 'Deprecation Warning: The "text" argument in the "enableProperty" function will be deprecated ' +
```

Comment:
Semantic: it is deprecated already (will become with your change). What will happen in the future is the removal of the function.  
  
To avoid saying exactly when we'll remove it, let's just say that it will be removed in a future release (without pointing out precisely when). I assume that it's going to be few months (iterations) but noting his written in stone.